html-skeleton/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+  <html>
+    <head>
+      <title>
+        HTML Skeletons
+      </title>
+    </head>
+    <body>
+      <p>
+        Hello, HTML!
+      </p>
+    </body>
+    <footer>
+
+    </footer>
```

Comment:
You don't need a footer, I don't mind extra html, but it needs to be in your body tags